packages/template-retail-react-app/config/default.js

```diff
@@ -24,9 +24,9 @@ module.exports = {
         commerceAPI: {
             proxyPath: `/mobify/proxy/api`,
             parameters: {
-                clientId: 'c9c45bfd-0ed3-4aa2-9971-40f88962b836',
-                organizationId: 'f_ecom_zzrf_001',
-                shortCode: '8o7m175y',
+                clientId: '574cf8f6-8536-4a39-acbb-8e7f1759f901',
+                organizationId: 'f_ecom_zzrf_023',
+                shortCode: 'kv7kzm78',
                 siteId: 'RefArchGlobal'
```

Comment:
will revert this change before merging

Reply:
@alexvuong will I need to setup the `zzrf_001` instance to work with shopper context as part of this ticket?

Reply:
@hajinsuha1 Yes, you will need to create sourceCode and tie it to a promotion and campaign for the sourceCode to work on instance zzrf_001